package.json

```diff
@@ -21,6 +22,8 @@
   "homepage": "https://github.com/webpack/tapable",
   "main": "lib/Tapable.js",
   "scripts": {
-    "test": "mocha --reporter spec"
+    "test": "mocha --reporter spec",
+    "release:create": "standard-version -i CHANGELOG.md -m \"Creating release %s\"",
+    "release:publish": "git push --follow-tags origin master; npm publish"
```

Comment:
@d3viant0ne sure you can include me.

I'm open for run release. I'll fix my changes and update all PR :)
We can move from there